### ISSUE TITLE
about training

### ISSUE THREAD

**Issue by ufukefe** [sniklaus/pytorch-pwc#17]
Hi, is it possible to train this network with your code on a dataset like Sintel?

**Comment by sniklaus** [comment]
The provided implementation is fully differentiable. As such, you just need to provide a data loader and implement a loss function. Feel free to fork this repository and see whether you can make it work. It would be nice to have a fork with a working training section that I could link to.
